tests/test_learndot.py

```diff
@@ -342,3 +344,38 @@ def test_rate_limit_is_retried(self, status_code, retry, mock_logger):
             mock_logger.warning.assert_called_with(retry)
         else:
             mock_logger.warning.assert_not_called()
+
+
+class ModelMock:
+    def exists(self):
+        print('is this ok?')
```

Comment:
nit: please remove this class -- I don't think you're using it anywhere?

Reply:
oops, ty for noticing, removed